infrastructure/cdn-in-a-box/enroller/Dockerfile

```diff
@@ -43,7 +43,7 @@ COPY ./traffic_ops/v4-client/ /go/src/github.com/apache/trafficcontrol/traffic_o
 COPY ./infrastructure/cdn-in-a-box/ /go/src/github.com/apache/trafficcontrol/infrastructure/cdn-in-a-box/
 
 WORKDIR /go/src/github.com/apache/trafficcontrol/infrastructure/cdn-in-a-box/enroller
-RUN set -o errexit -o nounset; \
+RUN set -o errexit; \
```

Comment:
We shouldn't remove `-o nounset`, though, since it found an error. We just need to move the definition for the build arg that is not properly defined.

Reply:
I did that on purpose instead of copying the arg down, because I was thinking "well it's only unset because I'm not debugging". That's not how that works.